### PR TITLE
Html tags are stripped from image captions.

### PR DESCRIPTION
**Pull request by eozan**
SDPA-294

### CODE REVIEW

**Comment by ioanpocol** [comment]
please rebase

File: scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -92,13 +92,25 @@
         return angular.extend(dest, _.pick(src, _.keys(CONTENT_FIELDS_DEFAULTS)));
     }
 
+    var htmlRegex = /(<([^>]+)>)/ig;
```
Comment (vied12) [inline scripts/superdesk-authoring/authoring.js]: can be move one row below to respect the scope

File: scripts/superdesk-authoring/authoring.js [hunk 2]
```diff
@@ -92,13 +92,25 @@
...
+    function removeHtmlRaw(content) {
```
Comment (vied12) [inline scripts/superdesk-authoring/authoring.js]: This can be moved into `RemoveTagsDirective`'s `link()` method

**Comment by naomiaro** [comment]
:+1:

**Pull request merged**